### PR TITLE
Updated with SdkName,SdkVersion,MachineName,ApplicationName.

### PR DESCRIPTION
**Pull request by venkattejagaddam**

### CODE REVIEW

File: src/LockstepApi.ts [hunk 1]
```diff
@@ -162,20 +166,67 @@ export class LockstepApi {
...
   /**
    * Construct headers for a request
    */
   private getHeaders(): any {
-    if (this.apiKey !== null) {
+    const hostName = _hostname();
+     if (hostName!==null) {
+       return {
+         
+         "SdkName": this.sdkName,
```
Comment (tspence) [inline src/LockstepApi.ts]: Can we change this a bit so we have less repetition?  Maybe something like this:

```
const hostName = _hostname();
let headers = { 
     "SdkName": this.sdkName,
     "SdkVersion": this.version,
     "ApplicationName": this.appName, 
     "MachineName": hostName, 
};
if (this.bearerToken !== null) {
  headers["Authorization"] = `Bearer ${this.bearerToken}`;
} else if (this.apiKey !== null) {
  headers["ApiKey"] = this.apiKey;
}
```

File: src/LockstepApi.ts [hunk 2]
```diff
@@ -190,8 +241,8 @@ export class LockstepApi {
       data: body,
       headers: this.getHeaders(),
     };
-    var result = await axios.default.request(requestConfig);
-    var response = new LockstepResponse<T>(result.status >= 200 && result.status < 300, result.status);
+    const result = await axios.default.request(requestConfig);
+    const response = new LockstepResponse<T>(result.status >= 200 && result.status < 300, result.status);
```
Comment (tspence) [inline src/LockstepApi.ts:222]: Good catch!

File: src/LockstepApi.ts (hunk 1 above)
Comment (venkattejagaddam) [reply src/LockstepApi.ts]: Yes.

**Comment by tspence** [comment]
LGTM!

**Pull request merged**